src/core/shm.lua

```diff
@@ -63,8 +63,7 @@ end
 
 function alias (name, target)
    mkdir(lib.dirname(resolve(name)))
-   assert(S.symlink(root.."/"..resolve(target), root.."/"..resolve(name)),
-          "shm alias failed")
+   assert(S.symlink(root.."/"..resolve(target), root.."/"..resolve(name)))
```

Comment:
Any reason for removing the assertion error message?

Reply:
S.symlink returns two values in an error case, but passing two args to `assert` truncates to the first value.  Removing the message allows the original error to propagate through.